registry.json

```diff
@@ -5419,5 +5419,19 @@
    "telegram": "Dinoaus73",
    "operatorName": "John Dean",
    "github": ""
+  },
+{
+   "ticker": "HASH",
+   "poolId": "b49a54a4bca1160af57d6314d4c1110b0c071db47eaaa08b619c4fa1",
+   "social": {
+     "website": "https://hashvalley.com/",
```

Comment:
There is almost no social media presence for this pool. Please can you add your twitter or other so we can see how you communicate with your delegates and attract new ones? This is just so we can verify you are not part of a pool group.